srcCpp/PerftestTransport.cxx

```diff
@@ -89,31 +89,46 @@ bool setAllowInterfacesList(
 {
     if (!transport.allowInterfaces.empty()) {
 
-        /*
-         * By default, if the transport is not set, it should be UDPv4, if it is not
-         * It means that we have modified the QOS, so we won't use the -nic param.
+        /* In these 2 specific cases, we are forced to add 2 properties, one
+         * for UDPv4 and another for UDPv6
          */
-        if (transport.transportConfig.kind == TRANSPORT_DEFAULT
```

Comment:
I just realized that I should still add this piece of code just in case:

```
        if (transport.transportConfig.kind == TRANSPORT_NOT_SET) {
            fprintf(stderr,
                    "%s Ignoring -nic/-allowInterfaces option\n",
                    classLoggingString.c_str());
            return true;
        }
```